src/master/develop/connection.md

```diff
@@ -51,8 +51,35 @@ immuClient.login("immudb", "immudb");
 :::
 
 ::: tab Python
-This feature is not yet supported or not documented.
-Do you want to make a feature request or help out? Open an issue on [Python sdk github project](https://github.com/codenotary/immudb-py/issues/new)
+
+```python
+from grpc import RpcError
+from immudb import ImmudbClient
+
+URL = "localhost:3322"  # immudb running on your machine
+LOGIN = "immudb"        # Default username
+PASSWORD = "immudb"     # Default password
+DB = b"defaultdb"       # Default database name (must be in bytes)
+
+
+def main():
+    client = ImmudbClient(URL)
+    # database parameter is optional
+    client.login(LOGIN, PASSWORD, database=DB)
```

Comment:
In immudb 1.2 we've introduced new session mechanism - @mabmayer is it used for this login functionality?

If not then we should promote the new session-based approach (that is necessary for opening SQL transactions)

Reply:
@byo @tomekkolo No session mechanism right now supported.